docs/site/Controller-generator.md

```diff
@@ -63,9 +63,10 @@ export class FooController {
 #### REST Controller with CRUD Methods
 
 If you select the REST Controller with CRUD Methods type, you will then be asked
-to select:
+to select or specify:
 
 - The model to use for the CRUD function definitions
+- The plural form of model to use for REST paths
```

Comment:
In my experience, the whole thing with plural model forms was rather confusing in LB 3.x. The connection from a plural model form to an HTTP path mapping is not obvious, plus there are examples where a plural form is confusing to people with a non-English primary language (e.g. a `Person` model is exposed at `/people`).

Also in LB4, it is possible to have a controller that's not tied to a single model (or any model at all), in which case the plural form makes even less sense.

I am proposing to be explicit about our intents in LB 4.x. When we need to decide how to map model's CRUD methods to a REST API, then ask the user about the base HTTP path to use. The fact that we use a plural form as the default path should be just our implementation detail.